itou/www/apply/views/process_views.py

```diff
@@ -284,6 +286,33 @@ def details_for_company(request, job_application_id, template_name="apply/proces
     return render(request, template_name, context)
 
 
+@require_POST
+@check_user(lambda user: user.is_employer)
+def shared_comment_for_company(request, job_application_id):
+    queryset = JobApplication.objects.is_active_company_member(request.user)
+    job_application = get_object_or_404(queryset, id=job_application_id)
+    is_locked = False
+    mode = "view"
+
+    if request.POST.get("acquire_lock"):
+        is_locked = job_application.acquire_shared_comment_lock(request.user)
+        mode = "view" if is_locked else "edit"
```

Comment:
Marrant on n'a pas la même logique (je m'attends à avoir `True` si le lock est acquis), du coup ici on aurait plutôt eu :

```suggestion
        lock_acquired = job_application.acquire_shared_comment_lock(request.user)
        mode = "edit" if lock_acquired else "view"
```